exercises/clock/clock_test.py

```diff
@@ -90,28 +90,28 @@ def test_add_more_than_two_days(self):
         self.assertEqual(str(Clock(1, 1) + 3500), '11:21')
 
     def test_subtract_minutes(self):
-        self.assertEqual(str(Clock(10, 3) + -3), '10:00')
+        self.assertEqual(str(Clock(10, 3) - 3), '10:00')
 
     def test_subtract_to_previous_hour(self):
-        self.assertEqual(str(Clock(10, 3) + -3), '10:00')
+        self.assertEqual(str(Clock(10, 3) - 3), '10:00')
```

Comment:
This test case and the next two seem to have inputs and outputs belonging to other test cases. Please fix them.

Reply:
Thanks for your meticulous review!

Reply:
This is still incorrect; I believe this test should be `self.assertEqual(str(Clock(10, 3) -30), '09:33')`

Please make this change and verify the other `test_subtract_*` test cases.